src/commands/profile.ts

```diff
@@ -0,0 +1,34 @@
+import { ApplicationCommandOptionType } from 'discord.js';
+
+export const profile: GlobalCommand = {
+	name: 'profile',
+	description: 'Responds with the Profile Picture of the mentioned user',
+	options: [
+		{
+			name: 'user',
+			description: 'The user to get the profile picture of',
+			type: ApplicationCommandOptionType?.User,
+		},
+	],
+	requiresGuild: false,
+	async execute({ interaction, options }) {
+		const user = interaction.user;
```

Comment:
The context includes the `user` property of the interaction, and a `reply` helper that handles errors and certain `CommandInteraction` edge cases.

```suggestion
	async execute({ user, options, reply }) {
```